realm/realm-library/src/main/java/io/realm/RealmObject.java

```diff
@@ -127,6 +127,10 @@ public final void deleteFromRealm() {
      * Checks if the RealmObject is still valid to use i.e. the RealmObject hasn't been deleted nor has the
      * {@link io.realm.Realm} been closed. It will always return false for stand alone objects.
      *
+     * <p>Note that a notification for Observer will <em>not</em> be triggered when a {@link RealmQuery}
```

Comment:
This sentence doesn't seem to match what is described here? https://github.com/realm/realm-java/issues/2158#issuecomment-174847313

We other methods we have embedded code samples so people can see how this can be used. Perhaps this was an idea here as well.